test.cpp

```diff
@@ -8,6 +8,7 @@ class Protein: public Model
 	    Protein(): Model("Proteins") {}
 	    
 	    IntegerField * proteinID = integerField ("Protein_ID");
+        CharField * proteinChain = charField ("Protein_Chain", 5); // Added for charField test
```

Comment:
There were still tab characters in this file. Apparently, I didn't think `test.cpp`  needed to be included when I switched tabs for spaces. It has been fixed now.